backtester/engine/setup.go

```diff
@@ -964,10 +965,13 @@ func setExchangeCredentials(cfg *config.Config, base *gctexchange.Base) error {
 			cfg.DataSettings.LiveData.ExchangeCredentials[i].Keys.PEMKey,
 			cfg.DataSettings.LiveData.ExchangeCredentials[i].Keys.OneTimePassword,
 		)
-		validated := base.AreCredentialsValid(context.TODO())
-		base.API.AuthenticatedSupport = validated
-		if !validated {
-			return fmt.Errorf("%v %w", base.GetName(), errInvalidCredentials)
+		creds, err := base.GetCredentials(context.TODO())
```

Comment:
```suggestion
		_, err := base.GetCredentials(context.TODO())
		if err != nil {
		          return err
		 }
```

And can drop the VerifyAPICredentials below since in this context `GetCredentials` will check them